src/Drupal/DrupalExtension/ServiceContainer/DrupalExtension.php

```diff
@@ -264,7 +264,7 @@ private function loadDrush(FileLoader $loader, ContainerBuilder $container, arra
             $config['drush']['alias'] = isset($config['drush']['alias']) ? $config['drush']['alias'] : false;
             $container->setParameter('drupal.driver.drush.alias', $config['drush']['alias']);
 
-            $config['drush']['binary'] = isset($config['drush']['binary']) ? $config['drush']['binary'] : 'drush';
+            $config['drush']['binary'] = isset($config['drush']['binary']) ? $config['drush']['binary'] : '';
```

Comment:
I think this should throw a `RuntimeException` if the binary path is not set. alternatively, it may default to `vendor/bin/drush`. But leaving this empty could trigger a cascade of issues that will be hard to debug.